### PR TITLE
Adding tgui formula

### PR DESCRIPTION
**Pull request by coldeasy**

### CODE REVIEW

**Comment by dunn** [comment]
Just a few audit failures:
```
==> audit problems
tgui:
 * Stable: Use GitHub tarballs rather than zipballs (url is https://github.com/texus/TGUI/archive/v0.7-alpha2.zip).
 * Please remove default template comments
```

**Comment by coldeasy** [comment]
@dunn fixed

File: Library/Formula/tgui.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Tgui < Formula
+  desc "GUI library for use with sfml"
+  homepage "http://tgui.eu"
+  url "https://github.com/texus/TGUI/archive/v0.7-alpha2.tar.gz"
```
Comment (DomT4) [inline Library/Formula/tgui.rb]: If this isn't stable it needs to live in https://github.com/Homebrew/homebrew-devel-only

**Comment by DomT4** [comment]
Closing per the stable notation I made a few weeks back. Thanks for your submission to Homebrew!

**Pull request closed without merging**